specifyweb/frontend/js_src/lib/components/FormMeta/CarryForward.tsx

```diff
@@ -142,6 +143,48 @@ export function CarryForwardConfig({
 const normalize = (fields: RA<string>): RA<string> =>
   Array.from(fields).sort(sortFunction(f.id));
 
+/**
+ * FEATURE: Extend this to all tables
+ * See https://github.com/specify/specify7/pull/4804
+ */
+function BulkCloneConfig({
+  table,
+}: {
+  readonly table: SpecifyTable;
+}): JSX.Element | null {
+  const [globalBulkEnabled, setGlobalBulkEnabled] = userPreferences.use(
+    'form',
+    'preferences',
+    'enableBukCarryForward'
+  );
+
+  const isBulkCarryEnabled = globalBulkEnabled.includes(table.name);
+
+  return tableValidForBulkClone(table) ? (
+    <Label.Inline className="rounded bg-[color:var(--foreground)]">
+      <Input.Checkbox
+        checked={isBulkCarryEnabled}
+        onChange={(): void =>
+          setGlobalBulkEnabled(toggleItem(globalBulkEnabled, table.name))
+        }
+      />
+      {formsText.bulkCarryForwardEnabled()}
+    </Label.Inline>
+  ) : null;
+}
+
+export const tableValidForBulkClone = (table: SpecifyTable): boolean =>
+  table === tables.CollectionObject &&
+  !(
+    tables.CollectionObject.strictGetLiteralField('catalogNumber')
+      .getUiFormatter()
+      ?.fields.some(
+        (field) =>
+          field.type === 'regex' ||
+          (field.type === 'numeric' && !field.canAutonumber())
+      ) ?? false
```

Comment:
This should be `true` instead of false? 

Also, why doesn't `getUiFormatter().canAutonumber()` not be enough?

Reply:
I used `false` instead of `true` to be more permissive rather than restrictive if `getUiFormatter` returns undefined. 

---

`getUiFormatter().canAutonumber()` would be enough to handle the second case (doing this would also handle the case for an AlphaNumeric field on a UIFormatter which contains an auto-incrementing section), but not if the Formatter contains a `regex` field, which is a case we we'd probably want to disable. 

i..e, they can have a regex type field in a formatter which would cause an error

```xml
 <field type="regex" size="6" minsize="4" value="[0-9]{4}[a-zA-Z]{0,2}" pattern="####[AA]"/>
```

Even if it's a case we shouldn't consider fixing (or just aren't right now for whatever reason), I think it's worth disabling in the UI if we _know_ it's going to always cause an error. 
Similar to https://github.com/specify/specify7/pull/5001#pullrequestreview-2116275954 and https://github.com/specify/specify7/pull/5001#issuecomment-2167039171

Errors can absolutely be beneficial to users and to us, but I think it's just a hindrance in this case. 
Let me know what you think. This was merged before I got back to this review, so hopefully not to late to make changes if needed.